test_app/tests/authentication/authenticator_plugins/test_google_oauth2.py

```diff
@@ -23,6 +25,45 @@ def test_oidc_auth_successful(authenticate, unauthenticated_api_client, google_o
     assert response.status_code == 200
 
 
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "key, secret, expected_status_code, expected_error",
+    [
+        (None, None, 400, {'KEY': ['This field may not be null.']}),
+        ('', None, 400, {'KEY': ['This field may not be blank.']}),
+        ('testgoogleoauth2', '', 400, {'SECRET': ['This field may not be blank.']}),
+        ('testgoogleoauth2', None, 201, {}),
+        ('testgoogleoauth2', "testgoogleoauth2_secret", 201, {}),
+    ],
+)
+def test_google_oauth2_callback_url_validation(
+    admin_api_client,
+    key,
+    secret,
+    expected_status_code,
+    expected_error,
+):
+    callback_url = "http://testserver/api/social/complete/ansible_base-authentication-authenticator_plugins-google_oauth2__google-oauth2-test/"  # noqa
```

Comment:
Instead of hard coding this here can we call `get_fully_qualified_url` so that if that function ever changes this test does not stop working?